ios/Classes/FlutterPDFView.m

```diff
@@ -109,8 +121,14 @@ - (instancetype)initWithFrame:(CGRect)frame
         PDFPage* page = [document pageAtIndex: defaultPage];
         [_pdfView goToPage: page];
 
+            // _pdfView.minScaleFactor = [args[@"setMinZoom"] doubleValue];
             _pdfView.minScaleFactor = _pdfView.scaleFactorForSizeToFit;
-            _pdfView.maxScaleFactor = 4.0;
+            _pdfView.maxScaleFactor = [args[@"setMaxZoom"] doubleValue]; // 4.0;
```

Comment:
Not proficient in this language, but does this code here fulfill the parameters set<Val>Zoom for iOS and you just forgot to update the doc comments on flutter_pdfview.dart?

Reply:
this is the correct title.